topics/introduction/tutorials/galaxy-intro-101-everyone/tutorial.md

```diff
@@ -0,0 +1,497 @@
+{: .hands_on}
+
+## Upload Iris dataset
+
+> ### {% icon hands_on %} Hands-on: Data upload
+>
+> 1. Import `iris.csv` from [Zenodo](https://zenodo.org/record/1319069/files/iris.csv) or from the data library (ask your instructor)
+>
+>    ```
+>    https://zenodo.org/record/1319069/files/iris.csv
+>    ```
+>
+>    {% include snippets/import_via_link.md %}
+>    {% include snippets/import_from_data_library.md %}
+>
+>    As default, Galaxy takes the link as name, so rename them.
+>
+> 2. Rename the dataset to `iris`
+>
+>    {% include snippets/rename_dataset.md %}
+>
+> 3. Check the datatype. The datatype of the iris dataset is `csv`. Change datatype
+> if it is different than `csv`.
+>
+>    {% include snippets/detect_datatype.md datatype="datatypes" %}
+>
+> 4. Add the tag `iris` to the dataset
+>
+>    {% include snippets/add_tag.md %}
+>
+{: .hands_on}
+
+# Pre-processing
+
+A pre-processing step can be required to proceed analysis. In this case, format convertion and header removal have to be processed.
+
+
+## Convert dataset **csv_to_tabular**
+
+> ### {% icon hands_on %} Hands-on: Converting dataset format 
+>    
+>    * Click on the {% icon galaxy-pencil %} **pencil icon** for the dataset to edit its attributes
+>    * In the central panel, click on the {% icon galaxy-gear %} **Convert** tab on the top
+>    * Select `Convert CSV to tabular`
+>    * Click the **Convert datatype** button
+>
+>    > ### {% icon comment %} Comment
+>    > A New dataset is created in the history panel. You can click on the dataset name to get an overview
+>    {: .comment}
+>
+> 2. Rename the dataset to `iris tabular`
+>
+>    {% include snippets/rename_dataset.md %}
+> 3. Add the tag `preprocessing` to the dataset
+>
+>    {% include snippets/add_tag.md %}
+> 4. Inspect the generated file by clicking on the {% icon galaxy-eye %} (eye) icon (**View data**)
+{: .hands_on}
+
+
+## Remove header
+
+> ### {% icon comment %} Search bar
+>
+> Different Galaxy servers may have tools available under different sections, therefore it is often useful to use the **search bar** at the top of the tool panel to find your tool.
+>
+> Additionally different servers may have multiple, similarly named tools which accomplish similar functions. For these tutorials, you should select precisely the one that is described. However, in your real analyses, you'll need to search among the various options to find the one that works for you.
+>
+{: .comment}
+
+> ### {% icon hands_on %} Hands-on: Removing header
+>
+> 1. **Remove beginning** {% icon tool %} with the following parameters:
+>
+>    - *Remove first*: `1` to remove the first line only.
+>    - *from*: {% icon param-file %}: select **iris tabular**
+>    - **Execute**
+>
+>    > ### {% icon comment %} Comment
+>    >
+>    > Use the **tools search box** to find **Removing beginning** {% icon tool %}. 
+>    {: .comment}
+>
+>    ![Settings for the `Remove beginning` tool](../../images/101_foreveryone_remove_beginning.png)
```

Comment:
Could you recreate this screenshot but zoom in more it is very hard to read any of the text on this one currently

Reply:
thanks!